src/lavinmq/shovel/shovel_store.cr

```diff
@@ -8,6 +8,13 @@ module LavinMQ
 
     forward_missing_to @shovels
 
+    def parse_uris(src_uri) : Array(URI)
+      uris = src_uri.as_s? ? [src_uri.as_s] : src_uri.as_a.map(&.as_s)
```

Comment:
This could probably be
```suggestion
      uris = src_uri.as_s? || src_uri.as_a.map(&.as_s)
```

Reply:
does not work for map, 
`undefined method 'map' for String (compile-time type is (Array(String) | String))`